tests/system/libraries/NvdaLib.py

```diff
@@ -385,7 +385,7 @@ def check_for_crash_dump(
 		"""
 		Checks if a crash.dmp exits and returns the crash dmp path if so
 		"""
-		crashPath = overridePath or _pJoin(_locations.logPath, "..", "nvda_crash.dmp")
+		crashPath = overridePath or _pJoin(_locations.profileDir, "nvda_crash.dmp")
```

Comment:
should this use `os.path.dirname` too? it makes sense to keep this relative to the logPath